modules/tinymce/src/plugins/link/main/ts/core/Actions.ts

```diff
@@ -39,11 +41,12 @@ const gotoSelectedLink = (editor: Editor) => (): void => {
 };
 
 const setupGotoLinks = (editor: Editor): void => {
-  editor.on('click', (e) => {
-    const link = getLink(editor, e.target);
-    if (link && VK.metaKeyPressed(e)) {
+  editor.on('click', (e: any) => {
```

Comment:
Why the any here. It is a proper type so `t.target` should exist.The metaKeyPressed takes keyboard like events and just checks for the metaKey, ctrlKey etc so that should also be on this event object.

- [ ] Remove any cast